cli-commands/forgae-shape/artifacts/ExampleContract.aes

```diff
@@ -0,0 +1,125 @@
+contract ToDoManager =
```

Comment:
This contract should not replace the old ExampleCOntract. This must be created on init only when shape command is called